combat-trainer.lic

```diff
@@ -1779,6 +1782,32 @@ class TrainerProcess
       bput(%w[weave bob circle].sample, 'roundtime', 'There is nothing else', 'Face what', 'You must be closer', 'You must be standing', 'Strangely, you don\'t feel like fighting right now', 'flying too high for you to attack') unless game_state.npcs.empty?
     when 'Analyze'
       analyze(game_state, 0)
+    when 'Locks'
+      return if game_state.loaded
```

Comment:
💡 I highly recommend extracting this into a helper method (leaving this function to mostly be managing control flow)